scripts/create-package/plop-templates-react/package.json.hbs

```diff
@@ -30,12 +29,15 @@
     "@fluentui/eslint-plugin": "",
     "@fluentui/react-conformance": "",
     "@fluentui/scripts": "",
+    "@testing-library/react": "",
+    "@testing-library/react-hooks": "",
     "@types/enzyme": "",
     "@types/enzyme-adapter-react-16": "",
     "@types/jest": "",
     "@types/react": "",
     "@types/react-dom": "",
     "@types/react-test-renderer": "",
+    "@types/webpack-env": "",
```

Comment:
I don't think we need webpack-env (and it's actually wrong that most other packages have it). That should only be for packages that use `require()` or `require.context`, which most packages shouldn't.
```suggestion
```